.docker/aiida-core-dev/aiida-clone-and-install.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+cd ~ || exit
+git clone https://github.com/aiidateam/aiida-core.git --origin upstream
+cd aiida-core || exit
```

Comment:
```suggestion
git clone https://github.com/aiidateam/aiida-core.git --origin upstream /home/aiida/aiida-core
```

Reply:
Is there a reason why you made this change?